pkgs/development/libraries/amtk/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl
+, pkgconfig, gnome3 }:
+let
+  version = "4.99.1";
+  pname = "amtk";
+in stdenv.mkDerivation {
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${gnome3.versionBranch version}/${pname}-${version}.tar.xz";
+    sha256 = "00fhvw5y638z584s8cfdslh47ngfzcgx4f0b0456sw8p754j3f8d";
+  };
+
+  passthru.updateScript = gnome3.updateScript { packageName = pname; };
+
+  nativeBuildInputs = [
+    gnome3.gtk.dev
+    pkgconfig
+  ];
+
+  doCheck = false;
+  # TODO: seems like tests need a display:
+  #   (./test-action-info-store:12578): Gtk-WARNING **: 12:30:20.100: cannot open display:
+  #   FAIL test-action-info-store (exit status: 133)
+
+  meta = with stdenv.lib; {
+    homepage = https://wiki.gnome.org/Projects/Amtk;
+    description = "Actions, Menus and Toolbars Kit for GTK+ applications";
+    maintainers = [ maintainers.manveru ];
+    license = licenses.lgpl21;
```

Comment:
That would be lgpl21Plus: https://gitlab.gnome.org/GNOME/amtk/blob/master/amtk/amtk-action-map.c#L8
The COPYING file usually contains only the text of the lower bound, since including the text of later versions is not possible.